spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/actions/MigrateTableSparkAction.java

```diff
@@ -102,6 +103,12 @@ public MigrateTableSparkAction dropBackup() {
     return this;
   }
 
+  @Override
+  public MigrateTableSparkAction withBackupTableName(String tableName) {
```

Comment:
We could implement Spark 3.4 for this PR and do a backport for other versions.

Reply:
Sure. Revert the Spark 3.3 commit to the previous in the latest commit.